src/app/pages/checkout-payment/checkout-payment/checkout-payment.component.ts

```diff
@@ -166,6 +177,10 @@ export class CheckoutPaymentComponent implements OnInit, OnChanges, OnDestroy {
     });
   }
 
+  disableNextStep(disableNextStep: boolean) {
```

Comment:
Remove the code to disable the button (also all other related code), better we have only an error message in case the user forgets to enter the code